doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -151,11 +151,6 @@ Traces details about the provider and engine configuration.
 
 Traces the ENGINE algorithm table selection.
 
-More precisely, functions like ENGINE_get_pkey_asn1_meth_engine(),
```

Comment:
this is removed by Neil, but rather twice than zero times https://github.com/openssl/openssl/pull/28556/files